### PR TITLE
feat(max zoom): add max zoom option to export

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

File: src/util/isBBoxResolutionValid.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
+const zoomToResolution: number[] = [
```
Comment (asafMasa) [inline src/util/isBBoxResolutionValid.ts:2]: i think the resolutions array should be in config file, what do you think? 
for if one will want to change the tile scheme

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍 good job

File: src/util/isBBoxResolutionValid.ts (hunk 1 above)
Comment (shimoncohen) [reply src/util/isBBoxResolutionValid.ts:2]: You are correct, this is a temporary fix. There is an issue discussing what you suggested. I think this fix should happen when we have more knowledge about the wanted schemes.

**Review by asafMasa — APPROVED** [review]
🥇 Good job

**Pull request merged**